contracts/contracts/oracle/OracleRouter.sol

```diff
@@ -75,49 +95,63 @@ abstract contract OracleRouterBase is IOracle {
     }
 }
 
-/* Oracle Router that denominates all prices in USD
- */
+// @notice Oracle Router that denominates all prices in USD
 contract OracleRouter is OracleRouterBase {
     /**
      * @dev The price feed contract to use for a particular asset.
      * @param asset address of the asset
      */
-    function feed(address asset)
+    function feedMetadata(address asset)
         internal
         pure
         virtual
         override
-        returns (address)
+        returns (address feedAddress, uint256 maxDataStaleness)
```

Comment:
In this PR, sometimes it's `maxDataStaleness` and sometimes it `maxStaleness`. Let's just run with the shorter of the two and do `maxStaleness` throughout.

Reply:
agreed